src/components/custom/_agreement-label.js

```diff
@@ -55,14 +71,36 @@ const AgreementLabel = ({ handleChangeCheckbox, isChecked, color }) => {
                     ]}
                 />
             </CheckboxSpan>
+            {additional_text && (
+                <AdditionalFlex color={color}>
+                    <Localize
+                        fontSize="var(--text-size-xs)"
+                        translate_text={additional_text}
+                        components={[
+                            <LocalizedLinkText
+                                key={0}
+                                type={additional_link_path}
+                                external="true"
+                                rel="noopener noreferrer"
+                                size="14px"
+                                color="red"
+                            />,
+                        ]}
+                    />
+                </AdditionalFlex>
+            )}
```

Comment:
This section should actually exist outside of the agreement label component. We wouldn't want users to click on this line and it triggers the checkbox to be selected. 

This won't need to be part of the reusable component and can be added directly to the subscribe component 😄